srcpkgs/msdfgen/patches/cmake-install.diff

```diff
@@ -0,0 +1,274 @@
+commit 31dda2d47417b8999b9c873d642fba9a30cd2f9d
```

Comment:
Does the build system work without this patch?

Reply:
No, there is no install step in the old cmake file

Reply:
Manually install them then.